public/assets/scripts/choices.js

```diff
@@ -306,7 +306,7 @@ function () {
 
     var passedElement = typeof element === 'string' ? document.querySelector(element) : element;
 
-    if (!(passedElement instanceof HTMLInputElement || passedElement instanceof HTMLSelectElement)) {
+    if (!((passedElement === null || passedElement === void 0 ? void 0 : passedElement.constructor.name) === 'HTMLInputElement' || (passedElement === null || passedElement === void 0 ? void 0 : passedElement.constructor.name) === 'HTMLSelectElement')) {
```

Comment:
This change is more complex than what you've done elsewhere, does it need to be?

Assuming they can all be the same, let's move this functionality out to a function in `utils.ts`:
```javascript
export const isHTMLInputOrSelectElement = (element: HTMLElement): boolean => {
  const constructorName = element?.constructor.name;
  if (constructorName === 'HTMLInputElement' || constructorName === 'HTMLSelectElement') {
    return true;
  }
  return false;
}
```